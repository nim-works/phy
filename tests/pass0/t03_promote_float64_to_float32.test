discard """
  output: "(Done 0.5)"
"""
(TypeDefs
  (ProcTy (Float 4)))
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 0) 0 (Locals)
    (List
      (Block (Params)
        (Return
          (Promote (Float 8) (Float 4) (FloatVal 0.5)))))))
