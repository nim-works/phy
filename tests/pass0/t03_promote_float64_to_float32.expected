.type t0 () -> float
.start t0 p0
  LdImmFloat 0.5
  Ret
.end
